assets/src/components/template-mode-option/index.js

```diff
@@ -75,59 +85,76 @@ function getTitle( mode ) {
+ * @param {boolean} props.initialOpen Whether the panel should be open when the component renders.
  */
-export function TemplateModeOption( { children, details, mode, previouslySelected = false } ) {
+export function TemplateModeOption( { children, details, initialOpen, labelExtra = null, mode, previouslySelected = false } ) {
 	const { editedOptions, updateOptions } = useContext( Options );
 	const { theme_support: themeSupport } = editedOptions;
 
-	const id = `template-mode-${ mode }`;
+	const id = getId( mode );
 
 	return (
-		<Selectable id={ `${ id }-container` } className="template-mode-option" selected={ mode === themeSupport }>
-			<label htmlFor={ id }>
-				<div className="template-mode-selection__input-container">
-					<input
-						type="radio"
-						id={ id }
-						checked={ mode === themeSupport }
-						onChange={ () => {
-							updateOptions( { theme_support: mode } );
-						} }
-					/>
-				</div>
-				<div className="template-mode-selection__illustration">
-					{ <Illustration mode={ mode } /> }
-				</div>
-				<div className="template-mode-selection__description">
-					<h2>
-						{ getTitle( mode ) }
-					</h2>
-					{ previouslySelected && (
-						<AMPInfo>
-							{ __( 'Previously selected', 'amp' ) }
-						</AMPInfo>
-					) }
-				</div>
-			</label>
+		<AMPDrawer
+			heading={ (
+				<label className="template-mode-option__label" htmlFor={ id }>
+					<div className="template-mode-selection__input-container">
+						<input
+							type="radio"
+							id={ id }
+							checked={ mode === themeSupport }
+							onChange={ () => {
+								updateOptions( { theme_support: mode } );
+							} }
+						/>
+					</div>
+					<div className="template-mode-selection__illustration">
+						{ <Illustration mode={ mode } /> }
+					</div>
+					<div className="template-mode-selection__description">
+						<h3>
+							{ getTitle( mode ) }
+						</h3>
+						{ previouslySelected && (
+							<AMPInfo>
+								{ __( 'Previously selected', 'amp' ) }
+							</AMPInfo>
+						) }
+						{ labelExtra && (
+							<div className="template-mode-selection__label-extra">
+								{ labelExtra }
+							</div>
+						) }
+					</div>
+				</label>
+			) }
+			hiddenTitle={ getTitle( mode ) }
+			id={ `${ id }-container` }
+			initialOpen={ 'boolean' === typeof initialOpen ? initialOpen : mode && themeSupport && mode === themeSupport }
+			selected={ mode === themeSupport }
+		>
 			<div className="template-mode-selection__details">
-				<p>
-					<span dangerouslySetInnerHTML={ { __html: details } } />
-					{ ' ' }
-					{ /* @todo Temporary URL. */ }
-					<a href="http://amp-wp.org" target="_blank" rel="noreferrer">
-						{ __( 'Learn more.', 'amp' ) }
-					</a>
-				</p>
+				{ details && (
+					<p>
+						<span dangerouslySetInnerHTML={ { __html: details } } />
+						{ ' ' }
+						{ /* @todo Temporary URL. */ }
+						<a href="http://amp-wp.org" target="_blank" rel="noreferrer">
```

Comment:
@westonruter Will these be updated soon?

Reply:
Probably we'll update at RC1.